internal/testing/releasehelper.go

```diff
@@ -156,25 +156,33 @@ func runOnGomod(path string, reqHandler reqHandlerFunc) {
 	}
 }
 
+func gomodAddReplace(path string) {
```

Comment:
Nit: I would consider dropping the `reqHandlerFunc` type. It is only used in one place and just adds a level of indirection.

Reply:
Done